### PR TITLE
Positioning and Floating Elements (Microverse Project)

### PR DESCRIPTION
**Pull request by umairarshadbutt**
This pull Request for Review
Positioning and Floating Elements (Microverse Project)

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,710 @@
...
+  float: right;
+}
+
+.article-content-on-left {
+  flex: 55%;
+  width: 600px;
+  padding: 20px;
+  margin: 10px;
+
+  /* border-style: dotted; */
```
Comment (BrittanyBlake) [inline css/style.css:76]: Please remove commented code. 
Screen Shot 2020-06-13 at 1 50 24 PM

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,710 @@
...
+  border-left: 1px solid #e2e2e2;
+  padding-left: 11px;
+}
+
+/* .flex-container > div {
+  background-color: #f1f1f1;
+  height: 200px;
+  margin: 15px;
+} */
+
```
Comment (BrittanyBlake) [inline css/style.css:234]: Please remove commented code

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,710 @@
...
+    left: 30%;
+  }
+}
+
+.wrapper-container {
+  width: 600px;
+  margin-left: auto;
+  margin-right: auto;
+
+  /* background-color: aqua */
```
Comment (BrittanyBlake) [inline css/style.css:383]: Please remove commented code

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,710 @@
...
+  width: 80%;
+}
+
+.services ul {
+  font-size: 1rem;
+}
+
+.services a {
+  padding-top: 2px !important;
+}
```
Comment (BrittanyBlake) [inline css/style.css:356]: Try not to use important to override styles. Instead, try to use CSS specificity

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,710 @@
...
+
+.menu-content {
+  text-align: center;
+}
+
+.more-posts-flex-container {
+  display: flex;
+  flex-wrap: wrap-reverse;
+
+  /* background-color: DodgerBlue; */
```
Comment (BrittanyBlake) [inline css/style.css:545]: Please remove commented code

File: css/style.css [hunk 6]
```diff
@@ -0,0 +1,710 @@
...
+  width: 935px;
+}
+
+.sidebar-editors-pick {
+  flex: 45%;
+  padding: 20px;
+  margin: 10px;
+  width: 1100px;
+
+  /* border-style: dotted; */
```
Comment (BrittanyBlake) [inline css/style.css:708]: Please remove commented code

File: css/style.css [hunk 7]
```diff
@@ -0,0 +1,710 @@
...
+  color: rgb(18, 18, 18);
+  font-size: 1.0875rem;
+  text-decoration: none;
+}
+
+.more-posts-flex-container-for-sidebar {
+  display: flex;
+  flex-wrap: wrap-reverse;
+
+  /* background-color: DodgerBlue; */
```
Comment (BrittanyBlake) [inline css/style.css:592]: Please remove commented code

File: index.html [hunk 8]
```diff
@@ -0,0 +1,706 @@
...
+                <img class="space-image" src="img/cosmos-superJumbo.jpg" alt="cosmos-superJumbo" />
+                <p class="img-desc">
+                    Alan Guth was one of the first physicists to hypothesize the existence of inflation, which explains
+                    how the universe expanded so uniformly and so quickly in the instant after the Big Bang 13.8 billion
+                    years
+                    ago.Credit...Rick Friedman for The New York Times
+                </p>
+            </div>
+            <!-- <div style="width: 500px;">Description</div>
+                <div style="width: 400px;">Article</div> -->
```
Comment (BrittanyBlake) [inline index.html:59]: Please remove commented code

File: index.html [hunk 9]
```diff
@@ -0,0 +1,706 @@
...
+                                    <img class="more-article-image-for-sidebar" src="img/extra_posts/editor-picked-post3.jpg" alt="post1">
+                                    <div class="more-article-name-for-sidebar">
+                                        <h3> <a href="#">‘I Learned That Day That a Block Is Actually Pretty Long, East to
+                                                West’</a> </h3>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+            </div>
+        </div>
```
Comment (BrittanyBlake) [inline index.html:221]: Please use the `<aside>` tag here.

File: css/style.css (hunk 1 above)
Comment (umairarshadbutt) [reply css/style.css]: Removed
Thanks for the review and helps me to guide

File: css/style.css (hunk 2 above)
Comment (umairarshadbutt) [reply css/style.css]: Removed commented code

File: css/style.css (hunk 3 above)
Comment (umairarshadbutt) [reply css/style.css]: Removed commented code

File: css/style.css (hunk 4 above)
Comment (umairarshadbutt) [reply css/style.css]: Override style is removed

File: css/style.css (hunk 5 above)
Comment (umairarshadbutt) [reply css/style.css]: Removed commented code

File: css/style.css (hunk 6 above)
Comment (umairarshadbutt) [reply css/style.css]: Removed commented code

File: css/style.css (hunk 7 above)
Comment (umairarshadbutt) [reply css/style.css]: Removed commented code

File: index.html (hunk 8 above)
Comment (umairarshadbutt) [reply index.html]: Removed commented code

File: index.html (hunk 9 above)
Comment (umairarshadbutt) [reply index.html]: Now I use the `<aside>` tag

File: css/style.css [hunk 10]
```diff
@@ -0,0 +1,717 @@
...
+  margin-right: auto;
+  width: 1180px;
+}
+
+.more-posts {
+  flex: 75%;
+  padding: 20px;
+  margin: 10px;
+
+  /* border-style: dotted; */
```
Comment (BrittanyBlake) [inline css/style.css:479]: Please remove the commented code.

File: css/style.css [hunk 11]
```diff
@@ -0,0 +1,717 @@
...
+}
+
+.most-popular {
+  flex: 25%;
+  padding: 20px;
+  margin: 10px;
+
+  /* border-style: dotted; */
```
Comment (BrittanyBlake) [inline css/style.css:487]: Please remove the commented code.

File: index.html [hunk 12]
```diff
@@ -0,0 +1,755 @@
...
+<html lang="en">
+
+<head>
+    <meta charset="utf-8" />
+    <link href="css/style.css" rel="stylesheet" type="text/css" />
+    <link href="js/script.js" rel="javascript" type="text/js">
+    <title>Space Ripples Reveal Big Bang’s Smoking Gun</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+    <!-- <link rel="icon" href="img/favicon.png" type="image/gif" /> -->
```
Comment (BrittanyBlake) [inline index.html:11]: Please remove the commented code.

Comment (umairarshadbutt) [reply index.html]: done

File: css/style.css (hunk 11 above)
Comment (umairarshadbutt) [reply css/style.css]: done

File: css/style.css (hunk 10 above)
Comment (umairarshadbutt) [reply css/style.css]: done

File: index.html [hunk 13]
```diff
@@ -0,0 +1,747 @@
...
+                    how the universe expanded so uniformly and so quickly in the instant after the Big Bang 13.8 billion
+                    years
+                    ago.Credit...Rick Friedman for The New York Times
+                </p>
+            </div>
+        </div>
+    </section>
+
+    <section class="wrapper-container">
+        <span>By <a style="text-decoration: none;" href="#">Dennis Overbye</a></span>
```
Comment (HADDADSOHAIB) [inline index.html:73]: Inline style here

File: index.html [hunk 14]
```diff
@@ -0,0 +1,747 @@
...
+        <br />
+        <div class="date-socials">
+            <div class="date-of-published"><span>March 17, 2014</span></div>
+            <div class="social-icons">
+                <i class="fa fa-facebook"></i>
+                <i class="fa fa-twitter"></i>
+                <i class="fa fa-envelope"></i>
+                <i class="fa fa-share"></i>
+                <i style="background-color: #fff;" class="fa fa-bookmark-o fa-2x fontawesome-bookmark"></i>
```
Comment (HADDADSOHAIB) [inline index.html:82]: Inline style here

Comment (umairarshadbutt) [reply index.html]: done

File: index.html (hunk 13 above)
Comment (umairarshadbutt) [reply index.html]: done

**Pull request merged**